ui/src/test/java/edu/wpi/grip/ui/PaletteTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.wpi.grip.ui;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.util.Modules;
+import edu.wpi.grip.core.AdditionOperation;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.Step;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.core.events.StepAddedEvent;
+import edu.wpi.grip.util.GRIPCoreTestModule;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+import org.testfx.util.WaitForAsyncUtils;
+
+import javax.inject.Inject;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class PaletteTest extends ApplicationTest {
+
+    @Inject private EventBus eventBus;
+
+    private Step step;
+    private final GRIPCoreTestModule testModule = new GRIPCoreTestModule();
+
+    @Override
+    public void start(Stage stage) throws Exception {
```

Comment:
> ![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [A method/constructor shouldnt explicitly throw java.lang.Exception](https://www.codacy.com/app/WPIRoboticsProjects/GRIP/file/2056769179/issues/source?bid=2943107&fileBranchId=3130834#l34)

It doesn't need to. If there are any exceptions that can be thrown explicitly state them.